docs/networking.md

```diff
@@ -9,9 +9,14 @@ help you out.
 
 
 {% include /style/icon.html type="info" %}
-For Raspberry Pi platforms with no display, you must use the wired Ethernet port
-to connect for the first time. You can set up additional connections using the
-`connmanctl` command through a remote SSH session.
+For Raspberry Pi platforms with no display 
+([to be able to use Brickman](http://lechnology.com/2016/05/adding-a-display-to-brickpi/)), 
```

Comment:
OK... I agree. Truth is I was a bit hesitant myself for the same reasons.   
I suppose I should have discussed it before pushing a new commit.  I'm new to the procedures of PRs.  I'll strip out this change.  

Do I just push a new commit again? (